### PR TITLE
Add absolute chart to benchmark details

### PR DESCRIPTION
**Pull request by lqd**
This PR adds an absolute chart to the benchmark detail panel, to complement the relative chart:
- the relative chart easily shows cyclical noise, and the common magnitude seen on a given benchmark
- the absolute chart easily shows whether a change was noise and has returned to steady state, or is real and has stayed the same since the commit.

Screenshot

image

### CODE REVIEW

File: site/frontend/src/pages/compare/compile/table/benchmark-detail.vue [hunk 1]
```diff
@@ -359,16 +381,20 @@ onMounted(() => renderGraph());
     flex-grow: 1;
   }
 }
+.columns + .columns {
```
Comment (Kobzol) [inline site/frontend/src/pages/compare/compile/table/benchmark-detail.vue:384]: I have been doing webdev for 12+ years and this is the first time I'm seeing this :eyes: Bloody CSS. I would probably just add a new class, like `.graphs`, and add the style to that, to make it more obvious/readable.

**Review by Kobzol — COMMENTED** [review]
This is great, I like the horizontal layout and the fact that the graphs are both visible at once.

For me, they render below each other though:
image

File: site/frontend/src/pages/compare/compile/table/benchmark-detail.vue (hunk 1 above)
Comment (lqd) [reply site/frontend/src/pages/compare/compile/table/benchmark-detail.vue]: Done

**Comment by lqd** [comment]
I also reduced the graph width a bit so that both should fit in a single row consistently, which should fix the issue in your screenshot.

**Comment by lqd** [comment]
Actually incredible. I pushed a `nowrap`, and hope it works for you because 3rd time's the charm.

**Review by Kobzol — APPROVED** [review]
Now it works, thanks! Just remove https://github.com/rust-lang/rustc-perf/compare/b873b613af29f44ab5be13466b82b765af690c8e..627d02e9a5f7d000f251f9f55f8c1dc54b3806cb#diff-813a534b1cbe87a3c3d6370efa9e279cec7c8083308f493d6dd95e12d9feab81R1, and otherwise LGTM.

**Pull request merged**